### DISCUSSION TITLE
make display-messages lsp config true by default

### DISCUSSION THREAD

**Discussion by adsick** [helix-editor/helix#10354] [General]
[This guides](https://docs.helix-editor.com/configuration.html?highlight=lsp#lsp) how to turn on lsp status messages, but one may have a wrong workspace setup, lsp will produce errors and user won't see them  which is bad.

**Comment by matoous** [comment]
I would say `display-messages` is too verbose for it to be on by default. Maybe the config could have severity value instead (`error`, `warn`, `info`) and only filtered messages could be displayed.

**Comment by adsick** [comment]
Kinda agree but still think that the messages are **useful** so it is better to see them than not to see.
